docs/references/javascript/types/use-organization-params.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: useOrganizationParams
+description: Parameters for the `useOrganization()` hook.
+---
+
+# `useOrganizationParams`
+
+Parameters for the [`useOrganization()`](/docs/references/react/use-organization) hook.
```

Comment:
How come we didn't go for [such format](https://docs-preview-332.clerkpreview.com/docs/references/react/use-organization-list#common-paginated-params) (params in the same page as table ) ?